### PR TITLE
Allow cert duration to be configurable

### PR DESCRIPTION
**Pull request by doanac**
This PR consists of 2 small commits to clean up some linter warnings then the real change a757a43

### CODE REVIEW

**Comment by doanac** [comment]
@mwasilew - this should allow us to configure the default certificate duration.

File: cmd/main.go [hunk 1]
```diff
@@ -83,7 +85,8 @@ func main() {
 		log.Fatal().Err(err).Msg("Unable to create tls cert handler")
 	}
 
-	svcHandler := est.NewStaticServiceHandler(est.NewService(rootCert, caCert, caKey))
+	duration := time.Hour * 24 * time.Duration(*durationDays)
```
Comment (mwasilew) [inline cmd/main.go:88]: Do I read it correctly that shortest possible duration is 1 day?

Comment (vkhoroz) [reply cmd/main.go:88]: hmm... interesting question.

@doanac I think we can allow `cert-duration-days` to accept either a float or a generic _time duration_ format.
https://pkg.go.dev/time#ParseDuration

Comment (doanac) [reply cmd/main.go]: i've changed it to accept golang's time.Duration format:

 https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/time/format.go;l=1585-1588

**Pull request merged**